modules/p2-profile/product/pom.xml

```diff
@@ -1293,7 +1293,7 @@
                                 </feature>
 				<feature>
                                     <id>org.wso2.carbon.social.feature.group</id>
-                                    <version>1.1.0</version>
+                                    <version>1.2.0</version>
```

Comment:
Since we are changing the version on related "social" features, better extract them pom properties e.g. <carbon.social.feature.version>1.2.0</carbon.social.feature.version>